misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -35,11 +35,13 @@ import java.nio.file.Paths
 import java.time.Duration
 import java.util.Properties
 import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.concurrent.thread
 import kotlin.reflect.KClass
 import kotlin.streams.toList
 
 const val VITESS_VERSION = "sha256:3cef0e042dee2312e5e0d80c56a6bf1581b52063ad0441551241f6785b110c38"
+const val CONTAINER_NAME_PREFIX = "cash-vitess-testing"
```

Comment:
Call it `misk-vitess-testing` instead. We want misk to be generally useful even outside of Square.

Reply:
Oops, I had it as misk at one point